src/EnergyPlus/RoomAirModelManager.cc

```diff
@@ -2676,7 +2697,7 @@ namespace RoomAir {
                         std::string const &EquipName, // Equipment Name
                         std::string &SupplyNodeName,  // Supply node name
                         std::string &ReturnNodeName,  // Return node name
-                        int TotNumEquip,              // how many of this equipment type
+                        int EquipIndex,               // Equipment index
```

Comment:
There is no need for TotNumEquip. EquipIndex is added to get index for each type of zone equipment